pbxd/v3/views.py

```diff
@@ -0,0 +1,24 @@
+from . import v3
+from flask import request, abort
+from ..app import logger
+from ..app import pbx
+from flask import current_app as app
+
+
+@v3.route('/<pbx_name>', methods=['POST'])
+def pbx_command(pbx_name):
+    if pbx_name != app.config["PBX_NAME"]:
+        logger.error('connected to the wrong PBX: {} != {}'.format(pbx_name, app.config["PBX_NAME"]))
+        abort(500)
```

Comment:
I don't really understand why the configuration is like this. If you have to run one instance of the app per PBX, why not make this the root path of the blueprint, and have the `/uw01` path part be provided by the proxy server forwarding to the app? I understand that the `/v3/` part comes first, but if we run more than one instance of this in MCI, the paths will have to look something like `/pbxd/uw01/v3/uw01`. Or am I missing something?

Anyway it would make more sense to me to tweak how this works and support configuring multiple PBXes for a single app instance, unless there's some reason that's not feasible.

Reply:
As it is now the nginx proxy routes to http://host/v3/n1, http://host/v3/n2, etc. With APPLICATION_ROOT conf set to / it all maps correctly.

This is a check that nginx and the deployed app are configured to match, and prevents a misconfiguration of the proxy to provide connect a client to the wrong pbx.

Thinking a bit more about the REST API it's probably not normal to have the resource name come after the version of it, so I should probably reverse that so it's /uw01/v3.

Does that sound better to you?

Reply:
I think version number before or after both work. There's no advantage to either. I prefer version number first so they all group together.